### DISCUSSION TITLE
How can I disable dark mode on forms in a Livewire component?

### DISCUSSION THREAD

**Discussion by oliverbj** [filamentphp/filament#7601] [Help]
In the documentation, I can only figure out how to disable the dark mode for a panel...

**Comment by danharrin** [comment]
`->darkMode(false)`

**Reply by rasul-chopurov** [reply]
> ->darkMode(false)

how to disable livewire table filament ? ->darkMode(false)not work

**Reply by rasul-chopurov** [reply]
> `->darkMode(false)`

Method Filament\Tables\Table::darkMode does not exist.

**Reply by danharrin** [reply]
Send your Tailwind config file

**Reply by rasul-chopurov** [reply]
> Send your Tailwind config file
darkmode false or class not work, in panel providers already not work

import preset from '../../../../vendor/filament/filament/tailwind.config.preset'

export default {
    presets: [preset],
    darkMode: 'false', 
    content: [
        './app/Filament/**/*.php',
        './resources/views/filament/**/*.blade.php',
        './vendor/filament/**/*.blade.php',
        './vendor/jaocero/radio-deck/resources/views/**/*.blade.php',
    ],
    theme: {
        extend: {
            colors: {
                white: '#F6F5F3',
            },
        }
    }
}

**Reply by danharrin** [reply]
`darkMode: 'false',` is not a valid option AFAIK, so I think it is falling back to the default "system" setting in Tailwind. You can just remove this option, and as long as there is not a `dark` class in your app it will not be in dark mode.

**Reply by rasul-chopurov** [reply]
> `darkMode: 'false',` is not a valid option AFAIK, so I think it is falling back to the default "system" setting in Tailwind. You can just remove this option, and as long as there is not a `dark` class in your app it will not be in dark mode.

Can you tell me correctly how to deactivate this?

**Reply by danharrin** [reply]
Remove the `darkMode: 'false',` line and recompile your CSS

**Reply by rasul-chopurov** [reply]
> Remove the `darkMode: 'false',` line and recompile your CSS

How can I disable black mode for tables and forms?

**Reply by danharrin** [reply]
Don't include a `dark` class in your HTML. Filament's default is to look for that class and apply dark mode there.

**Reply by rasul-chopurov** [reply]
> Don't include a `dark` class in your HTML. Filament's default is to look for that class and apply dark mode there.

how to globally disable dark mode lookup in filament ?
appservice provider add?

**Reply by danharrin** [reply]
It's nothing to do with PHP, it's in the Tailwind JS config and the HTML...

The only time you control dark mode with PHP is in a panel, when Filament controls the HTML.

**Reply by Kepsondiaz** [reply]
> `->darkMode(false)`

https://github.com/filamentphp/filament/discussions/7601#discussioncomment-10012609

**Comment by Kepsondiaz** [comment]
you can add darkMode: 'false' in your tailwind.config.js file . in this way, dark mode will be disabled in all your filament form .

`
export default {
        content: [
    
        ],
    
        darkMode: 'false',
    
        theme: {
            extend: {
                fontFamily: {
                    sans: ['Figtree', ...defaultTheme.fontFamily.sans],
                },
            },
        },
    
        plugins: [forms, typography],
};

`
